system/core.go

```diff
@@ -62,6 +62,19 @@ func (application *Application) Init(APISecret, baseURL, cookieSecret string,
 	application.APISecret = APISecret
 }
 
+var funcMap = template.FuncMap{
+	"times": func(a, b float64) float64 {
+		return a * b
+	},
+	"netname": func(network string) string {
+		net := strings.ToLower(network)
+		if strings.HasPrefix(net, "testnet") {
```

Comment:
#445 actually added this functionality already. 

https://github.com/decred/dcrstakepool/blob/625bcd23ec3901d7b732903b9255a1a03104e0d2/controllers/main.go#L170-L179

https://github.com/decred/dcrstakepool/blob/625bcd23ec3901d7b732903b9255a1a03104e0d2/controllers/main.go#L192

---

And we also have other places where the URL is being constructed in the template:

https://github.com/decred/dcrstakepool/blob/625bcd23ec3901d7b732903b9255a1a03104e0d2/views/tickets.html#L179

---

We should probably try and standardise all of this and re-use/refactor the existing code rather than adding new code. Perhaps we could construct the dcrdata URL once in `config.go` and just pass the full URL around, rather than constructing the URL in multiple places.

Reply:
The reason for having hit here is that in some routes, `{{$.Network}}` is not passed through `(*MainController).getNetworkName` first.  On such routes, the "testnet3" is needed, so the number can't be stripped from it.  The solutions are: (1) add another data field passed to the template with both versions of the network name, or (2) strip it on the fly in template execution.